rotkehlchen/tests/api/test_metadata.py

```diff
@@ -0,0 +1,40 @@
+from typing import TYPE_CHECKING
+
+import requests
+
+from rotkehlchen.chain.ethereum.airdrops import AIRDROPS
+from rotkehlchen.chain.ethereum.defi.protocols import DEFI_PROTOCOLS
+from rotkehlchen.tests.utils.api import api_url_for, assert_proper_response_with_result
+
+if TYPE_CHECKING:
```

Comment:
Perhaps you can move other metadata/mapping api endpoints tests here too. Such as the `test_query_supported_chains`.

Any other @yabirgb ?